Untiering is now a synchronous operation (#2404)
timescale/docs: use-timescale/data-tiering/untier-data.md

## Untier data in a chunk

> Untiering chunks is a synchronous process and will occur when the `untier_chunk` procedure is called. 

Reviewer: ```suggestion
Untiering chunks is a synchronous process that occurs when the `untier_chunk` procedure is called. 
```
